src/platform/user/profile/utilities/index.js

```diff
@@ -84,7 +84,7 @@ export function mapRawUserDataToState(json) {
       last,
     },
     verified,
-    vet360: isVAProfileServiceConfigured()
+    vapContactInfo: isVAProfileServiceConfigured()
```

Comment:
🔥 🔥 This is the meat of this PR's change. Pretty much everything else is an update to deal with this change in how contact info is stored in Redux.